src/context/FoxEthProvider/FoxEthProvider.tsx

```diff
@@ -29,7 +29,14 @@ import type { Nullable } from 'types/common'
 const moduleLogger = logger.child({ namespace: ['FoxEthContext'] })
 
 export type FoxFarmingEarnOpportunityType = {
-  unclaimedRewards: string
+  /**
+   * @deprecated Here for backwards compatibility until https://github.com/shapeshift/web/pull/3218 goes in
+   */
+  unclaimedRewards?: string
+  stakedAmountCryptoPrecision?: string
+  rewardsAmountCryptoPrecision?: string
+  underlyingToken0Amount?: string
+  underlyingToken1Amount?: string
```

Comment:
Unused currently, to be consumed in the big girl PR https://github.com/shapeshift/web/pull/3218